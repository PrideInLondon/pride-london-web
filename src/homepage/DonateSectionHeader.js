import React from 'react'
import { Header, Title, Description } from './DonateSectionHeader.styles'

const DonateSectionHeader = () => (
  <Header>
    <Title>Keep Pride in London going</Title>
    <Description>
      We rely on your support to keep going. Please donate to power our vital
      work and help us address the needs of LGBT+ people through our Unity Fund.
    </Description>
  </Header>
)

export default DonateSectionHeader
