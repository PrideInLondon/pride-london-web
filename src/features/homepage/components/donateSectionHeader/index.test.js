import React from 'react'
import { shallow } from 'enzyme'
import DonateSectionHeader from '.'

describe(DonateSectionHeader.name, () => {
  it('volunteer snapshot', () => {
    const wrapper = shallow(<DonateSectionHeader />)
    expect(wrapper).toMatchSnapshot()
  })
})
