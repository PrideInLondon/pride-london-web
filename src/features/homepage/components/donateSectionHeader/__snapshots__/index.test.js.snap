// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DonateSectionHeader volunteer snapshot 1`] = `
<styles__Header>
  <styles__Title>
    Keep Pride free for everyone
  </styles__Title>
  <styles__Description>
    We rely on your support to keep Pride free and accessible to all. Please consider making a small donation to keep our community marching!
  </styles__Description>
</styles__Header>
`;
